encoding/thrift/thriftrw-plugin-yarpc/client.go

```diff
@@ -64,10 +64,14 @@ type Interface interface {
 //
 // 	client := <$pkgname>.New(dispatcher.ClientConfig("<lower .Service.Name>"))
 func New(c <$transport>.ClientConfig, opts ...<$thrift>.ClientOption) Interface {
-	return client{c: <$thrift>.New(<$thrift>.Config{
-		Service: "<.Service.Name>",
-		ClientConfig: c,
-	}, opts...)}
+	return client{
+		c: <$thrift>.New(<$thrift>.Config{
+			Service: "<.Service.Name>",
+			ClientConfig: c,
+		}, opts...),<if .Parent>
+		<$parentPath := printf "%s/%sclient" .ParentModule.ImportPath (lower .Parent.Name)>
```

Comment:
this is hardcoded here and below in the client struct, should we move the logic to generate the name into a common place?